tendermint-lite/src/lib.rs

```diff
@@ -1,4 +1,2 @@
 pub mod requester;
-pub mod state;
 pub mod store;
-pub mod threshold;
```

Comment:
state and threshold modules are not needed anymore as the provided structs `TrustedState<C, H>` and `TrustThresholdFraction` provide exactly the same functionality.